components/automate-cli/pkg/verifyserver/server/api/v1/opensearchbackups3.go

```diff
@@ -0,0 +1,45 @@
+package v1
+
+import (
+	"fmt"
+
+	"github.com/chef/automate/components/automate-cli/pkg/verifyserver/models"
+	"github.com/chef/automate/components/automate-cli/pkg/verifyserver/response"
+	"github.com/gofiber/fiber/v2"
+)
+
+func (h *Handler) CheckOSBackupS3(c *fiber.Ctx) error {
+
+	req := new(models.S3BackupDetails)
+
+	if err := c.BodyParser(req); err != nil {
+		errString := fmt.Sprintf("Invalid request for S3 backup check: %v", err.Error())
+		h.Logger.Error(fmt.Errorf(errString))
+		return fiber.NewError(fiber.StatusBadRequest, err.Error())
+	}
+
+	if !requestValidator(*req) {
+		errString := fmt.Sprintf("Invalid request body for S3 backup check")
```

Comment:
error is used for two time can be taken common or constant

Reply:
done